unit_testing/test_labels.py

```diff
@@ -139,11 +139,28 @@ def test_compute_mean_squared_error():
     for x, y, z, _ in src.getNonZeroCoordinates():
         if z < 5:
             ref.data[x, y, z+2] = src.data[x, y, z]
-
     mse = sct_labels.compute_mean_squared_error(src, ref)
     assert mse == 1.1547005383792515
 
 
+def test_compute_mean_squared_error_warning(caplog):
+    src = fake_3dimage_sct()
+    ref = src.copy()
+
+    src.data[0, 0, 0] = 95
+
+    sct_labels.compute_mean_squared_error(src, ref)
+    assert 'False positive value for label 95.0' in caplog.text
+    assert 'False negative value for label 111.0' in caplog.text
```

Comment:
It might be good to try to fetch the value `111.0` programmatically. That way, if `fake_3dimage_sct` changes in the future, the test won't break.

Reply:
Agreed.

Reply:
It has been corrected (and the message has changed per your suggestion)